src/NSwag.Tests/CommandLine/AspNetCoreToSwaggerCommandLineTest.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace NSwag.Tests.CommandLine
+{
+    [TestClass]
+    public class AspNetCoreToSwaggerCommandLineTest
```

Comment:
I'll try and get more combination of tests (tfm \ Mvc versions). This uses what was already in your repository.